errors.go

```diff
@@ -164,55 +89,64 @@ func Errorf(format string, vals ...interface{}) error {
 	// if so, use that stacktracer in the StackErr.
 	var st StackTracer
 	if errors.As(err, &st) {
-		return StackErr{
-			Err:         err,
+		return stackErr{
+			err:         err,
 			stackTracer: st,
 		}
 	}
-	return StackErr{
-		Err:   err,
+	return stackErr{
+		err:   err,
 		trace: buildStackTrace(),
 	}
 }
 
 // Unwrap exposes the error wrapped by StackErr.
-func (se StackErr) Unwrap() error {
-	return se.Err
+func (se stackErr) Unwrap() error {
+	return se.err
 }
 
 // Error is the marker interface for an error, it returns the wrapped error or an empty string if there is no
 // wrapped error.
-func (se StackErr) Error() string {
-	if se.Err == nil {
+func (se stackErr) Error() string {
+	if se.err == nil {
 		return ""
 	}
-	return se.Err.Error()
+	return se.err.Error()
 }
 
 // Format controls the optional display of the stack trace. Use %+v to output the stack trace, use %v or %s to output
 // the wrapped error only, use %q to get a single-quoted character literal safely escaped with Go syntax for the wrapped
 // error.
-func (se StackErr) Format(s fmt.State, verb rune) {
+func (se stackErr) Format(s fmt.State, verb rune) {
 	switch verb {
 	case 'v':
 		if s.Flag('+') {
 			fmt.Fprintf(s, "%+v\n", se.Unwrap())
-			trace, _ := Trace(se, StandardFormat)
-			fmt.Fprintf(s, "%s", strings.Join(trace, "\n"))
+			trace, _ := Trace(se, PanicFormat)
```

Comment:
This whole PR is basically exclusively breaking changes. In particular, removing types and variables from the exported interface would break software that uses them.

Since `serrors` is pre-1.0, I was planning on making this a minor version bump (`v0.3.1` -> `v0.4.0`). I wanted to prevent people from relying on the things they already shouldn't have been relying on based on our documentation, and I was hoping to do that now because presumably our public adoption is virtually zero: https://github.com/search?q=github.com%2Fmorningconsult%2Fserrors&type=code

If you think it's likely that code is going to break because of this change, and that it's important for us to keep a backward compatibility on this code already even though we're not yet 1.0, I'd probably discard this change and tag a 1.0 version to reflect that. But I'd much rather take this being pre-1.0 and the low likelihood of this change breaking real software as an opportunity to prune our interface down to something we're comfortable committing to for when we eventually do go 1.0.